src/ur5e_arm.hpp

```diff
@@ -109,7 +109,7 @@ class UR5eArm : public Arm, public Reconfigurable {
                          const std::vector<vector6d_t>& p_a,
                          const std::vector<double>& time,
                          bool use_spline_interpolation_);
-    void read_and_noop();
+    bool read_joint_keep_alive();
```

Comment:
updated this to return a bool to match [nick's pr](https://github.com/viam-modules/universal-robots/pull/25), but decided to skip handling the bool since the other pr already implements that behavior.